src/main/java/seedu/address/model/Deadline.java

```diff
@@ -14,11 +14,16 @@ public class Deadline {
     /**
      * Represents an empty deadline.
      */
-    private static class EmptyDeadline extends Deadline {
-        private static final Deadline EMPTY_DEADLINE = new EmptyDeadline();
+    public static class EmptyDeadline extends Deadline {
+        public static final Deadline EMPTY_DEADLINE = new EmptyDeadline();
 
         private EmptyDeadline() {
-            super("");
+            super("2022-05-03");
```

Comment:
A more unique default string might help to distinguish the empty singletons from actual ones later on in `Storage`